webapp/src/ts/services/header-tabs.service.ts

```diff
@@ -69,11 +73,40 @@ export class HeaderTabsService {
       if (settings.header_tabs[tab.name].icon && settings.header_tabs[tab.name].icon.startsWith('fa-')) {
         tab.icon = settings.header_tabs[tab.name].icon;
       }
+
       if (settings.header_tabs[tab.name].resource_icon) {
         tab.resourceIcon = settings.header_tabs[tab.name].resource_icon;
       }
     });
 
     return this.tabs;
   }
+
+  async getAuthorizedTabs(settings:any = {}): Promise<HeaderTab[]> {
+    const tabs = this.get(settings);
+    const results = await Promise.all(tabs.map(tab => this.authService.has(tab.permissions)));
```

Comment:
Let's give this a suggestive name like `tabAuthorization`. 
Also ... it's a shame we don't have an authService bulk permissions check, that would receive a list of permissions and return an array, except not read user roles and settings for every group.

Reply:
Rename added!